### ISSUE TITLE
apply('forget, facts()) gives Lisp error

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#1339]
Imported from SourceForge on 2024-07-04 09:53:03
Created by **[willisbl](https://sourceforge.net/u/willisbl/)** on 2021-08-09 11:34:16
Original: https://sourceforge.net/p/maxima/bugs/3825

---

I've observed this with 5.43.2 + SBCL 1.4.14 + wxMaxima & with a recent CL build using Clozure CL (using command line Maxima)

~~~
(%i1) declare(F, increasing)$

(%i2) facts();
(%o2) [kind(F, increasing)]

(%i3) apply('forget, facts());

simplifya: operator is neither an atom nor a lambda expression: (((KIND %ATANH $INCREASING)) <junk deleted>

~~~

The output of %o2 is OK, but we shouldn't get a CL error with the input %i3

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-04 09:53:05
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2021-08-15 06:26:58
Original: https://sourceforge.net/p/maxima/bugs/3825/#069c

---

- **status**: open --> closed

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-04 09:53:08
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2021-08-15 06:26:58
Original: https://sourceforge.net/p/maxima/bugs/3825/#2dd7

---

Fixed by commit 68898be.
